src/zenml/cli/secret.py

```diff
@@ -79,7 +79,7 @@ def secret(ctx: click.Context) -> None:
         "--schema",
         "-s",
         "secret_schema_type",
-        default=ARBITRARY_SECRET_SCHEMA_TYPE,
+        default="arbitrary",  # TODO: Place in a constant outside secret module
```

Comment:
can't you just move `ARBITRARY_SECRET_SCHEMA_TYPE` to `zenml.constants` or something ?

Reply:
moved it!